randovania/games/fusion/logic_database/Sector 2 TRO.txt

```diff
@@ -1452,23 +1459,29 @@ Extra - room_id: [47]
   > Door to Central Shaft
       Trivial
   > Door to Nettori Save Room
-      Have Any Jump Upgrade
+      Any of the following:
+          Space Jump or Movement (Advanced) or Wall Jump (Intermediate)
+          Hi-Jump and Movement (Intermediate)
 
 > Door to Collapsed Shaft; Heals? False
   * Layers: default
   * Open Hatch to Collapsed Shaft/Door to Puyo Palace
   * Extra - door_idx: (109,)
   > Door to Central Shaft
-      Have Any Jump Upgrade
+      Trivial
 
 > Door to Central Shaft; Heals? False
   * Layers: default
   * Open Hatch to Central Shaft/Door to Puyo Palace
   * Extra - door_idx: (110,)
   > Door to Collapsed Shaft
-      Space Jump
+      Any of the following:
+          Space Jump
+          Wall Jump (Beginner) and Can Single Walljump
   > Door to Nettori Save Room
-      Space Jump
+      Any of the following:
+          Space Jump
+          Wall Jump (Beginner) and Can Single Walljump
```

Comment:
are you just tossing all SWJ at WJ beginner for now?

Reply:
no, this was me being careless and not thinking taht this is high enough where it should be intermediate.